bullet_train-api/lib/tasks/bullet_train/api_tasks.rake

```diff
@@ -146,5 +146,103 @@ namespace :bullet_train do
         )
       end
     end
+
+    desc "Create `api_title` and `api_description` translations"
+    task create_translations: :environment do
```

Comment:
@newstler Is this a task that's intended to be used by end users/developers? If so, we should probably document it somewhere. And maybe mention in the PR description that people should run it?

Reply:
I don't exactly know. Why and when would people need to use it? Maybe that can help us figure out where to surface it.

Reply:
They only need to run it once, to update all the old translations they have. As there is no documentation for the bt-api gem, probably there should be a post-install massage? I have some thoughts about bt-api README, but right now seems post-install message should work just fine, I'll try to add it.

Reply:
To be honest I would even run this task automatically after gem install, do you think this is an acceptable approach?

Reply:
I'm not a _huge_ fan of the idea of running it automatically. _Maybe_ if it's extremely fast to run. But otherwise it's going to be taking time every time someone runs `bundle install` whether locally or in CI/CD.

I think a post-install message would be better than automatically running.

Reply:
Added post-install message in `bullet_train-api/bullet_train-api.gemspec`.

I will try to create README for bt-api soon in a separate PR, and will mention this (and other) rake tasks there.